lua/nvim-surround/buffer.lua

```diff
@@ -117,6 +118,25 @@ M.set_operator_marks = function(motion)
     M.set_mark(">", visual_marks[2])
 end
 
+-- Gets extmark position for the current buffer.
+---@param extmark integer The extmark ID number.
+---@return position @The position of the extmark in the buffer.
+---@nodiscard
+M.get_extmark = function(extmark)
+    local ns = vim.api.nvim_create_namespace("nvim-surround")
+    local pos = vim.api.nvim_buf_get_extmark_by_id(0, ns, extmark, {})
+    return { pos[1] + 1, pos[2] + 1 }
+end
+
+-- Creates an extmark for the given position.
+---@param pos position The position in the buffer.
+---@return integer @The extmark ID.
+---@nodiscard
+M.set_extmark = function(pos)
+    local ns = vim.api.nvim_create_namespace("nvim-surround")
```

Comment:
I was under the impression that the namespace function was almost like a hashing function that would be deterministic, but if its e.g. monotonic increasing then yes I should globally make one somewhere (e.g. `M.namespace` or something)

Reply:
Do check this though, I'm on my phone right now with no way to check what I write about this😬

Reply:
Will do (tomorrow), I'm also just on my phone 🤣

Reply:
![image](https://github.com/kylechui/nvim-surround/assets/48545987/75aa20c7-50b5-418b-a2c9-91787d0d08b6)
Seems like it should be fine either way, but I'll probably create a local object since it semantically makes more sense; thanks again for pointing it out